Yefer-garcia/assets/style/style.css

```
@@ -0,0 +1,67 @@
+@import url("https://fonts.googleapis.com/css2?family=Oswald:wght@500&family=Roboto:wght@500&display=swap");
+
+/* Centrado */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+body {
+  display: flex;
+  justify-content: center;
+}
+.revista {
+  align-items: center;
+  background-image: url(/assets/img/shakira.jpg);
```

Please use relative path, if you run this in the browser you'll find it does not work

```suggestion
  background-image: URL(../img/shakira.jpg);
```